.travis.yml

```diff
@@ -32,6 +33,8 @@ matrix:
     gemfile: gemfiles/activerecord_6.0.0.gemfile
   - rvm: 2.4.2
     gemfile: gemfiles/activerecord_6.0.0.gemfile
+  - rvm: 2.7.0
+    gemfile: gemfiles/activerecord_4.2.0.gemfile
```

Comment:
The matrix excludes Rails 4.2 testing to prevent the following error.

```console
NoMethodError:
  undefined method `new' for BigDecimal:Class
```

- https://travis-ci.org/CanCanCommunity/cancancan/jobs/635119862
- https://travis-ci.org/CanCanCommunity/cancancan/jobs/635119863

The behavior of `BigDecimal.new` between Ruby 2.6 and Ruby 2.7 has the following differences.

```console
% ruby -rbigdecimal -ve 'BigDecimal.new(42)'
ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-darwin17]
-e:1: warning: BigDecimal.new is deprecated; use BigDecimal() method instead.
```

```console
% ruby -rbigdecimal -ve 'BigDecimal.new(42)'
ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-darwin17]
Traceback (most recent call last):
-e:1:in `<main>': undefined method `new' for BigDecimal:Class (NoMethodError)
```

Rails 4.2 is no longer maintained and will not be backported to support Ruby 2.7.
https://guides.rubyonrails.org/maintenance_policy.html

Therefore, this PR excludes the matrix of the combination of Ruby 2.7 and Rails 4.2.